ehr/resources/web/ehr/window/CreateTaskFromRecordsWindow.js

```diff
@@ -346,7 +346,7 @@ Ext4.define('EHR.window.CreateTaskFromRecordsWindow', {
         EHR.Utils.createTask({
             initialQCState: 'Scheduled',
             existingRecords: existingRecords,
-            taskRecord: {date: date, assignedTo: assignedTo, category: 'task', title: title, formType: this.formType},
```

Comment:
same question as above. my knowledge of usage here is out of date, but the original scheme was to create the dataset records where 'date' is the date that work is expected to happen. did something change here?

Reply:
discussion above covers this